scheduledJobs/giveApples.js

```diff
@@ -0,0 +1,271 @@
+const cron = require('node-cron');
+const Promise = require('promise');
+const { fetchAllMessagesByChannelSince, batchItems, sendAppleEmbed } = require('../common/discordutil');
+const { DateTime } = require('luxon');
+const { MessageAttachment } = require('discord.js');
+const { guildId } = require('../config.json');
+
+// For Testing in BA Code
+const greenAppleRoleId = '964598327775744070';
+const redAppleRoleId = '964597638047612969';
+const rollCallRoleId = '841115841852080129';
+// apple-log channel
+const resultsChannelId = '1033147239343861760';
+// every 5 min
+const cronSchedule = '*/5 * * * *';
```

Comment:
will remove before merge to main. leaving in case anyone else wants to use for testing.